azure-kusto-data/azure/kusto/data/helpers.py

```diff
@@ -48,9 +48,13 @@ def dataframe_from_result_table(table: "Union[KustoResultTable, KustoStreamingRe
 
     columns = [col.column_name for col in table.columns]
     frame = pd.DataFrame(table.raw_rows, columns=columns)
+    print(frame.dtypes)
+    print(frame.infer_objects().dtypes)
```

Comment:
Are these prints in the non-test code intentional?

Reply:
Thanks!